pkg/volume/emptydir/quota.go

```diff
@@ -84,6 +94,11 @@ func (cr *realQuotaCommandRunner) RunApplyQuotaCommand(fsDevice string, quota re
 	return "", stderr.String(), err
 }
 
+func (cr *realQuotaCommandRunner) RunMountOptionsCommand(dir string) (string, string, error) {
+	outBytes, err := exec.Command("mount").Output()
```

Comment:
Perhaps a better way to do this would be to have `mockQuotaCommandRunner` contain a `dir` field that you use in the mock `RunMountOptionsCommand` implementation